src/PrestaShopBundle/Resources/config/services/adapter/order_return.yml

```diff
@@ -21,10 +21,47 @@ services:
       - '@prestashop.adapter.order_return.repository.order_return_repository'
       - '@prestashop.adapter.order_return_state.repository.order_return_state_repository'
 
-  prestashop.adapter.order_return.repository.order_return_repository:
-    class: 'PrestaShop\PrestaShop\Adapter\OrderReturn\Repository\OrderReturnRepository'
+  PrestaShop\PrestaShop\Adapter\OrderReturn\Repository\OrderReturnRepository:
+    autowire: true
     arguments:
-      [ '@prestashop.adapter.order_return.validator.order_return_validator' ]
+      - '@doctrine.dbal.default_connection'
+      - '%database_prefix%'
+      - '@prestashop.adapter.order_return.validator.order_return_validator'
+
+  prestashop.adapter.order_return.repository.order_return_repository:
+    alias: PrestaShop\PrestaShop\Adapter\OrderReturn\Repository\OrderReturnRepository
+
+  PrestaShop\PrestaShop\Adapter\OrderReturn\Validator\OrderReturnValidator:
 
   prestashop.adapter.order_return.validator.order_return_validator:
-    class: 'PrestaShop\PrestaShop\Adapter\OrderReturn\Validator\OrderReturnValidator'
+    alias: PrestaShop\PrestaShop\Adapter\OrderReturn\Validator\OrderReturnValidator
+
+  PrestaShop\PrestaShop\Adapter\OrderReturn\QueryHandler\GetOrderDetailCustomizationsHandler:
+    autowire: true
+    arguments:
+      - '@=service("prestashop.adapter.legacy.context").getContext().language.id'
+      - '@prestashop.adapter.order.repository.order_repository'
+    tags:
+      - name: tactician.handler
+        command: 'PrestaShop\PrestaShop\Core\Domain\OrderReturn\Query\GetOrderDetailCustomizations'
+
+  PrestaShop\PrestaShop\Core\Domain\OrderReturn\QueryHandler\GetOrderDetailCustomizationsHandlerInterface: '@PrestaShop\PrestaShop\Adapter\OrderReturn\QueryHandler\GetOrderDetailCustomizationsHandler'
+
+  PrestaShop\PrestaShop\Adapter\OrderReturn\CommandHandler\UpdateOrderReturnStateHandler:
+    autowire: true
+    tags:
+      - name: tactician.handler
+        command: 'PrestaShop\PrestaShop\Core\Domain\OrderReturn\Command\UpdateOrderReturnStateCommand'
+
+  PrestaShop\PrestaShop\Core\Domain\OrderReturn\CommandHandler\DeleteProductFromOrderReturnHandler:
+    autowire: true;
+    tags:
+      - name: tactician.handler
+        command: 'PrestaShop\PrestaShop\Core\Domain\OrderReturn\Command\DeleteProductFromOrderReturnCommand'
+
+  PrestaShop\PrestaShop\Core\Domain\OrderReturn\CommandHandler\BulkDeleteProductFromOrderReturnHandler:
+    autowire: true;
+    tags:
+      - name: tactician.handler
+        command: 'PrestaShop\PrestaShop\Core\Domain\OrderReturn\Command\BulkDeleteProductFromOrderReturnCommand'
+    public: false
```

Comment:
you correctly added public:false here, but all the services above are missing this :arrow_up: